README.adoc

```diff
@@ -166,11 +173,12 @@ The [hotspot=maven-failsafe-plugin]`maven-failsafe-plugin` artifact runs your Ar
 
 Lastly, specify the [hotspot=liberty-maven-plugin]`liberty-maven-plugin`
 configuration that defines your Open Liberty runtime configuration.
-The `configure-arquillian` goal configures your Arquillian container.
-This goal automatically retrieves core server parameters from the `liberty-maven-plugin` configuration
-and enables you to customize your Arquillian container configuration directly in your POM file if necessary.
-You can learn more about the `configure-arquillian` goal in its
-https://github.com/WASdev/ci.maven/blob/master/docs/configure-arquillian.md[documentation^].
+When the application runs in the Arquillian Liberty managed container, 
+the name of the war file will be used as the context root of the application.
+Through the [hotspot=arquillianProperties]`arquillianProperties` configuration, 
+you can pass the context root information to the application and customize the container.
+You can learn more about the `arquillianProperties` configuration in the
+https://github.com/OpenLiberty/liberty-arquillian/blob/master/liberty-managed/README.md#configuration[Arquillian Liberty Managed Documentation^].
```

Comment:
Suggested update:

When the application runs in an Arquillian Liberty managed container, 
the name of the war file is used as the context root of the application.
You can pass context root information to the application and customize the container by using the [hotspot=arquillianProperties]`arquillianProperties` configuration.
To learn more about the `arquillianProperties` configuration, see the https://github.com/OpenLiberty/liberty-arquillian/blob/master/liberty-managed/README.md#configuration[Arquillian Liberty Managed documentation^].

Reply:
updated as suggest